Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -250,7 +255,7 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
       /// - Note: Generated files must be manually added to your application target. The generated
       /// schema types files will be namespaced with the value of your configuration's
       /// `schemaNamespace` to prevent naming conflicts.
-      case embeddedInTarget(name: String)
+      case embeddedInTarget(name: String, accessModifier: AccessModifier = .internal)
```

Comment:
It might seem odd that the default is `.internal` but if you look at the tests that have changed there were tests that named `xyz_withPublicModifier` and `xyz_noPublicModifier`. `withPublicModifier` was used for the tests related to a `.swiftPackageManager` module type and `noPublicModifier` was used for tests related to `.embeddedInTarget` module type. Since we weren't specifying any access modifier all the generated code would have implicitly had `internal` access level.

The caveat to this is the namespace enum, which we used to always generate as `public` - that was an error though and not aligned with the rest of the generated code for a `.embeddedInTarget` module type.